gnat2goto/driver/tree_walk.adb

```diff
@@ -204,6 +207,12 @@ package body Tree_Walk is
    function Do_Op_Not (N : Node_Id) return Irep
    with Pre => Nkind (N) in N_Op;
 
+   procedure Do_Package_Declaration (N : Node_Id)
```

Comment:
I seemed to have introduced a number of procedures starting with Do_* which is not in accordance with the naming convention according to https://github.com/xbauch/gnat2goto/pull/2#discussion_r249712062.

I don't think its worth changing now because it should mean re-ordering the declarations again to keep them in alphabetical order.  If it is ok with everyone merge with these names now and I will sort out the naming conventions when I do the next PR.

Reply:
@tjj2017 I have a small number of queries that are also just minor nitpicks / request for clarification (often just due to my poor knowledge of ADA) that I think should not get in the way of merging what is a great body of work. 
I'll ask about these separately - if additional comments are helpful then they can be in a separate minor PR.